source/develop/release-management/features/network/manageiq_ovn.html.md

```diff
@@ -0,0 +1,140 @@
+
+## Detailed Description
+ManageIQ has an infra provider for oVirt. The feature adds automatic discovery of oVirt's network provider (OVN provider) once the infra provider is added/refreshed.
+ManageIQ users will be able to perform CRUD actions on networks, subnets and ports directly on the OVN provider without passing through oVirt.
+ManageIQ already supports automatic discovery of the OpenStack network provider (Neturon) and has all the functionality to communicate with it (CRUD actions for networks, subnets, ports and more).
+Since oVirt's OVN provider has 'neutron like' API, the purpose of the feature is to reuse the already existing ManageIQ code (fog-openstack, ManageIQ::Providers::Openstack::NetworkManager) to communicate with the oVirt's OVN provider.
+Since there are some gaps between the 'neutron like' API oVirt's OVN provider has and the desired API ManageIQ expects to get, some adjustments to the oVirt's OVN provider are done during the development process.
+
+## Requirements
+* Discovery of oVirt network provider on the infra provider add/refresh.
+* CRUD actions on the OVN provider entities (networks, subnets, ports).
+* Manual refresh of oVirt's network provider.
+* Automatic refresh of oVirt's network provider (I'm not sure if it works for OpenStack- Neutron)
+
+## Limitations
+* ManageIQ supports only one network provider for oVirt.
+* In oVirt there is no way to distinguish between Openstack network provider and OVN provider. Therefore, ManageIQ has no way to find out what provider is the OVN one.
+In case oVirt has more than one network provider, the first one will be used.
+* ManageIQ is using the first network provider returned by /ovirt-engine/api/openstacknetworkproviders. If there is more than one, it can cause strange behavior in ManageIQ (I'm not sure, but I guess the order of the returned network providers is not persistent. Therefore, each time ManageIQ will read the details of another provider).
+* Currently, ManageIQ doesn't support removal of the network provider.
+
+## Benefit to oVirt
+* The ui and the communication code with the network provider is already written in ManageIQ (for Openstack-Neutron). Adding it from scratch to oVirt will take much more time than reusing the code of ManageIQ,
+
+## Benefit to ManageIq
+* Adding more power to the oVirt's infra manager.
+* oVirt users that want to use OVN provider (more than the basic operations oVirt supports) will have to use ManageIQ.
+
+## Gaps with the OpenStack network provider
+
+### Gaps
+* Communication and authentication
+  * The OpenStack infra manager added to ManageIQ has the keystone URL. OpenStack infra manager communicates with the other OpenStack's services via the keystone. The keystone returns a service catalog with all the URLs and other information about the services. This information is used to communicate with the other services.
```

Comment:
OpenStack infra manager communicates with the other OpenStack's services via the keystone.

->
OpenStack infra manager discovers the other OpenStack services via Keystone.